.github/workflows/npm-cd.yml

```
@@ -22,7 +23,7 @@ on:
                 required: true
 
 concurrency:
-    group: npm-${{ github.head_ref || github.ref }}
+    group: node-${{ github.head_ref || github.ref }}-${{ toJson(inputs) }}
```

it may conflict with node CI
```suggestion
    group: node-cd-${{ github.head_ref || github.ref }}-${{ toJson(inputs) }}
```